src/services/ContributorAuditArgumentParser.ts

```diff
@@ -43,13 +50,13 @@ class ContributorAuditArgumentParser extends ArgumentParserBase {
     });
 
     this.addEnumArgument(this.argumentParser, "--scmType", ScmType, {
-      help: "Scm Type to use for the audit. Options: GitHub.",
+      help: "Scm Type to use for the audit. Options: GitHub, Bitbucket.",
       default: ScmType.GitHub,
       required: false,
     });
 
-    this.argumentParser.add_argument("--organizationName", {
-      help: "Organization name to use for the audit.",
+    this.argumentParser.add_argument("--userName", {
```

Comment:
this param parse should be in the provider rather than as a generic param - also use "username" rather than "userName"